### DISCUSSION TITLE
Do changed ratings sync?

### DISCUSSION THREAD

**Discussion by ztevo** [Taxel/PlexTraktSync#1410] [Q&A] [answered]
Firstly - thanks for a great, easy to use app.

I have noticed that if I change a rating in Trakt, it is not updated in Plex. In other words, shows/movies will only have the user rating synced once.

Is this expected behaviour?

For example - If I rate GoT 4 stars in Trakt, 4 stars will then sync to Plex. However, if I change my rating to 5 stars in Trakt, Plex remains 4 stars.

 From reading GitHub, nothing suggested changes should not be updated...

Thanks

**Comment by simonc56** [comment, marked as answer]
> Is this expected behaviour?

No, it's a bug.
I'll fix it with #1411

**Comment by ztevo** [comment]
Good to know, and thanks kindly for the quick response

**Comment by ztevo** [comment]
Hello sorry to bother. I can see the change [#1411 ](https://github.com/Taxel/PlexTraktSync/pull/1411) was delivered - Thank you.
I can't quite figure out how GitHub prod releases work, as nothing is showing in history - does this mean the fix is scheduled or should it already be live? I am running 0.25.15
Thanks

**Reply by simonc56** [reply]
I just released `0.25.16` which includes the fix. Enjoy !

**Reply by ztevo** [reply]
> I just released `0.25.16` which includes the fix. Enjoy !

Really great, can confirm already it works. Thanks again!
